app/repos/cdo.js

```diff
@@ -242,7 +259,7 @@ const getAllCdos = async (idStart, rowLimit) => {
     query.subQuery = false
   }
 
-  return await sequelize.models.dog.findAll(query)
+  return sequelize.models.dog.findAll(query)
```

Comment:
Are you convinced the await should be removed? If so, should it become a sync (not async) method?

Reply:
sequelize.models.dog.findAll(query) is a Promise
so this function will return a Promise
the await resolves the promise, and then the async function then turns the resolved promise into a Promise again.  You need to put an await after a return if there is a try catch block - otherwise the error will bubble straight down the stack.

Reply:
" Async functions always return a promise. If the return value of an async function is not explicitly a promise, it will be implicitly wrapped in a promise. "

we don't need it to be async, it could just be a function.  It's nice though to have it as an async function as it is clear then that it is asynchronous.  As far as I know the inverse of the above is true...ie - a promise won't be wrapped in another promise.